mlflow/server/js/src/experiment-tracking/components/CompareRunView.js

```diff
@@ -410,14 +411,30 @@ export class CompareRunView extends Component {
                 runDisplayNames={this.props.runDisplayNames}
               />
             </TabPane>
+            <TabPane
+              tab={
+                <FormattedMessage
+                  defaultMessage='Box Plot'
+                  description='Tab pane title for box plot on the compare runs page'
+                />
+              }
+              key='3'
+            >
+              <CompareRunBox
+                  runUuids={runUuids}
+                  runInfos={runInfos}
+                  paramLists={paramLists}
+                  metricLists={metricLists}
+              />
+            </TabPane>
             <TabPane
               tab={
                 <FormattedMessage
                   defaultMessage='Contour Plot'
                   description='Tab pane title for contour plots on the compare runs page'
                 />
               }
-              key='3'
+              key='4'
```

Comment:
```suggestion
              key='box-plot'
```

Reply:
Can we also update the keys in other tabs?

Reply:
Yeah sure